lib/command_runner.dart

```diff
@@ -142,14 +142,14 @@ class CommandRunner<T> {
   /// here to enable verbose logging before running the command.
   ///
   /// This returns the return value of [Command.run].
-  Future<T> runCommand(ArgResults topLevelResults) async {
-    var argResults = topLevelResults;
+  Future<T?> runCommand(ArgResults topLevelResults) async {
+    ArgResults? argResults = topLevelResults;
```

Comment:
This seems like we should be able to shift where the `!` is and make the variable non-nullable.

Reply:
https://github.com/dart-lang/args/pull/163